README.md

```diff
@@ -23,12 +23,22 @@ Accord Project is an open source, non-profit, initiative working to transform co
 
 ## Installation
 
-To build the documentation locally:
+To build the documentation locally (docusaurus v1):
```

Comment:
Now redundant and can be removed

Reply:
fixed in 4f8d81d971a4ee1142b87a86d8b1fda3c4fa8025